deepchecks/base/dataset.py

```diff
@@ -381,6 +382,41 @@ def __len__(self) -> int:
         """Return number of samples in the member dataframe."""
         return self.n_samples
 
+    def train_test_split(self,
+                         train_size: t.Union[int, float] = None,
+                         test_size: t.Union[int, float] = None,
+                         random_state: int = 42,
+                         shuffle: bool = True) -> t.Tuple[TDataset, TDataset]:
+        """Split dataset into random train and test.
+
+        Args:
+            train_size (float or int):
+                If float, should be between 0.0 and 1.0 and represent the proportion
+                of the dataset to include in the test split. If int, represents the
+                absolute number of test samples. If None, the value is set to the
+                complement of the train size. If ``train_size`` is also None, it will
+                be set to 0.25.
```

Comment:
Why not specifying in the method signature that 
train_size: t.Union[int, float] = 0.25?